src/components/CurrencyInputPanel/index.tsx

```diff
@@ -269,28 +327,42 @@ export default function CurrencyInputPanel({
                     <DoubleCurrencyLogo currency0={pair.token0} currency1={pair.token1} size={24} margin={true} />
                   </span>
                 ) : currency ? (
-                  <CurrencyLogo style={{ marginRight: '0.5rem' }} currency={currency} size={'24px'} />
+                  <CurrencyLogo style={{ marginRight: '2px' }} currency={currency} size={'24px'} />
                 ) : null}
                 {pair ? (
-                  <StyledTokenName className="pair-name-container">
+                  <StyledTokenName className="pair-name-container" phase0Flag={phase0FlagEnabled}>
                     {pair?.token0.symbol}:{pair?.token1.symbol}
                   </StyledTokenName>
                 ) : (
-                  <StyledTokenName className="token-symbol-container" active={Boolean(currency && currency.symbol)}>
+                  <StyledTokenName
+                    className="token-symbol-container"
+                    active={Boolean(currency && currency.symbol)}
+                    phase0Flag={phase0FlagEnabled}
+                  >
                     {(currency && currency.symbol && currency.symbol.length > 20
                       ? currency.symbol.slice(0, 4) +
                         '...' +
                         currency.symbol.slice(currency.symbol.length - 5, currency.symbol.length)
-                      : currency?.symbol) || <Trans>Select a token</Trans>}
+                      : currency?.symbol) || <Trans>Select token</Trans>}
                   </StyledTokenName>
                 )}
               </RowFixed>
-              {onCurrencySelect && <StyledDropDown selected={!!currency} />}
+              {onCurrencySelect && <StyledDropDown selected={!!currency} phase0Flag={phase0FlagEnabled} />}
             </Aligner>
-          </CurrencySelect>
+          </InputCurrencySelect>
         </InputRow>
+        {phase0FlagEnabled && !currency && (
+          <NoBalanceState>
+            <FiatRow phase0Flag={phase0FlagEnabled}>
+              <RowBetween>
+                <NoBalanceDash>-</NoBalanceDash>
```

Comment:
i think this should be done using borders rather than dashes

Reply:
it's a null state of a price though

Reply:
so it's a dash to replace text, otherwise would need a rly small div with a small border